Contributing.md

```diff
@@ -0,0 +1,125 @@
+# Planetary Software Organization Community Contributing Guide
+
+This document describes a very simple process suitable for activities
+in the Planetary Software Organization ecosystem, including the TSC
+itself.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an
+	issue or pull request.
+* A **Committer** is a subset of contributors who have been given
+	write access to the repository.
+* A **Technical Committee (TC)** is a group of committers representing
+	the required technical expertise to resolve rare disputes.
+	For the purpose of this repository, the Planetary Software
+	Technical Steering Committee is the TC.
+
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in
+doubt, log an issue, any additional policies about what to include
+will be provided in the responses. The only exception is a security
+disclosure which should be sent privately.
```

Comment:
We should add guidelines about how to do this. Maybe, contact the TC directly?

Reply:
I took a stab.  Take a look.